src/modules/credentials/repository/CredentialRecord.ts

```diff
@@ -98,6 +104,7 @@ export class CredentialRecord extends BaseRecord implements CredentialStoragePro
 
     return new CredentialInfo({
       claims,
+      attachments: this.attachments ? this.attachments : undefined,
```

Comment:
I think we can just use this.attachments. (as it will be undefined if it has not value)
```suggestion
      attachments: this.attachments,
```

Otherwise

```suggestion
      attachments: this.attachments ?? undefined,
```